.github/ISSUE_TEMPLATE/designer-bug-report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Designer Bug Report
+about: Create a bug report for problems in the Adaptive Cards designer
+title: "[Designer] Bug Title Here"
+labels: Bug, Triage-Needed, Area-Designer
+assignees: ''
+
+---
+
+# Host Application
+
+Please provide the name of the application that is rendering the card (ex: Bot Framework, Outlook, Teams, etc)
+
+# Target Version
+
+Please provide the Adaptive Cards SDK version (ex: 1.3, 1.4)
```

Comment:
Do we want SDK version or card schema version?

Reply:
Hmm good call, I don't think we need Target version or Host app in this bug type

Reply:
Good point, removed.